tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/Sqlite/SqliteDefinitions.cs

```diff
@@ -12,7 +12,7 @@
     AssemblyName = "Microsoft.Data.Sqlite",
     TypeName = "Microsoft.Data.Sqlite.SqliteCommand",
     MinimumVersion = "2.0.0",
-    MaximumVersion = "8.*.*",
+    MaximumVersion = "9.*.*",
```

Comment:
nit: we have a constant to make it easier to update these (not sure how I missed it originally :()
```suggestion
    MaximumVersion = SupportedVersions.LatestDotNet,
```

Reply:
I only added it in the .NET 9 update 🙂 Thanks!